src/rpc/server/scala/api/src/main/scala/com/ctc/omega_edit/SessionImpl.scala

```diff
@@ -220,7 +219,6 @@ private[omega_edit] class SessionImpl(p: Pointer, i: FFI) extends Session {
 private object Edit {
   def apply(op: => Long): Change.Result =
     op match {
-      case 0          => Change.Fail
```

Comment:
Why is this being removed?

Reply:
Because we shouldn't attempt to handle errors in this layer.  Let errors flow through and give the client the opportunity to handle them.  We can't really catch errors thrown in this layer, can we?  Best we can do is log atypical results and pass them through to the _client_ to decide what to do.